openapi/atlas-api-transformed.yaml

```diff
@@ -24614,30 +23983,20 @@ components:
         createdAt:
           type: string
           format: date-time
-          description: Date and time when someone created this export job. MongoDB Cloud
-            represents this timestamp in ISO 8601 format in UTC.
+          description: Date and time when a user or Atlas created the Export Job. MongoDB
+            Cloud represents this timestamp in ISO 8601 format in UTC.
           readOnly: true
         customData:
           type: array
           description: Collection of key-value pairs that represent custom data for the
-            metadata file that MongoDB Cloud uploads to the bucket when the
-            export job finishes.
+            metadata file that MongoDB Cloud uploads when the Export Job
+            finishes.
           items:
             $ref: "#/components/schemas/BackupLabel"
-        deliveryUrl:
-          type: array
-          description: 'One or more Uniform Resource Locators (URLs) that point to the
-            compressed snapshot files for manual download. MongoDB Cloud returns
-            this parameter when `"deliveryType" : "download"`.'
-          items:
```

Comment:
deliveryURL is now hidden, more context in https://github.com/10gen/mms/pull/110218